docs/field-details.asciidoc

```diff
@@ -3,6 +3,7 @@
 
 The `base` field set contains all fields which are at the root of the events. These fields are common across all types of events.
 
+[discrete]
```

Comment:
`discrete` is equivalent to `float`, correct? That's what I think I understand from the asciidoctor docs.

No need to change this, btw. If they're indeed equivalent, I actually prefer the word "discrete" over "float".

Reply:
Yes the two seem to function the same.

I came across a bullet point in the AsciiDoctor docs mentioning preference of `[discrete]` over `[float]` [here](https://asciidoctor.org/docs/asciidoc-asciidoctor-diffs/), as well as the Elastic doc's [chunking guidance](https://github.com/elastic/docs#chunking-selected-sections) suggesting `[discrete]` too.